cogs/admin.py

```diff
@@ -85,7 +122,7 @@ def check(r, u):
 **Guild name:** {guild.name}
 **Guild owner:** {await self.bot.fetch_user(guild.owner_id)}
 **Guild owner ID:** {guild.owner_id}
-**{len(guild.humans)}** humans & **{len(guild.bots)}** bots
+Approximately **N/A** members
```

Comment:
use `guild.member_count` here

Reply:
I attempted this, but that would just error out